hnn_core/gui/gui.py

```diff
@@ -1393,7 +1397,7 @@ def add_drive_widget(drive_type, drive_boxes, drive_widgets, drives_out,
 
             for idx, drive in enumerate(drive_widgets):
                 tab_name = drive['name']
-                if "Tonic" not in tab_name:
+                if "tonic" not in tab_name.lower():
```

Comment:
any reason to prefer not matching by case?

Reply:
Usually good practice to match on a standard case if it's something that can be variable. In this case the title of the drive can be passed in from an input file. That said...this if-condition should be using the drive _type_ not the drive _name_, as the _type_ is a standard set that's not variable like _title_.